src/lib/es2015.reflect.d.ts

```diff
@@ -87,6 +87,7 @@ declare namespace Reflect {
     /**
      * Sets the property of target, equivalent to `target[propertyKey] = value` when `receiver === target`.
      * @param target Object that contains the property on itself or in its prototype chain.
+     * @param value The value to set.

```

Comment:
It should come after `propertyKey`, not before

Reply:
Thanks for pointing out such a detailed question, I didn't pay attention to the order of the parameters, it's fixed now.